src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/hmi/vi_subscribe_vehicle_data_response_template.h

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
this shall be in vehicle info plugin

Reply:
@yang1070 this RPC is applicable only for HMI_DBUS_API that is not supported by SDL right now and as I know will be removed within 5.0 ( same version as for this changes)